src/main/resources/db/migration/courtcase/V2133__remove_duplicate_recrods.sql

```diff
@@ -0,0 +1,37 @@
+BEGIN;
+
+DROP TABLE IF EXISTS court_case_new;
+
+create table court_case_new as select * from court_case cc where cc.id  in (select max(id) from court_case cc2 group by cc2.case_id);
+
+ALTER TABLE court_case_new ADD PRIMARY KEY (id);
+
+ALTER TABLE court_case_new
+    ALTER COLUMN id
+        ADD GENERATED BY DEFAULT AS IDENTITY;
+
+ALTER TABLE ONLY court_case_new ALTER COLUMN deleted SET DEFAULT false;
```

Comment:
Good spot with this. very difficult one. Also add default as `now()` for created